### PR TITLE
Added Momentum Indicator and Strategy

### PR DESCRIPTION
**Pull request by Shawn8901**
Since there are a few questions about "simple" strategies (#898, #335 ) i have added the momentum indicator and a strategy based on that.
https://en.wikipedia.org/wiki/Momentum_(technical_analysis)

As Strategy i have used a strategy based on double momentum analysis, which i found on tradingview.com. If both momentums have the same result it sends buy/sell signal.

It also performs okayish on paper trade for short terms trades.
Simulation 

>end balance: 4543.31295643 (354.33%)
>buy hold: 752.83890672 (-24.72%)
>vs. buy hold: 503.49%
>13481 trades over 9 days (avg 1497.88 trades/day)
>win/loss: 3070/3671
>error rate: 54.45%

Paper-Trade for ~6h

>last balance: 1344.79165820 (34.47%)
>buy hold: 1027.19439233 (2.71%)
>vs. buy hold: 30.91%
>122 trades over 1 days (avg 122.00 trades/day)
>win/loss: 38/23
>error rate: 37.70%

### CODE REVIEW

File: lib/_codemap.js [hunk 1]
```diff
@@ -23,5 +23,6 @@ module.exports = {
   'slow_stochastic': require('./slow_stochastic'),
   'collection-service': require('./services/collection-service'),
   'cmf': require('./cmf'),
-  'bollinger': require('./bollinger')
+  'bollinger': require('./bollinger'),
+ 'momentum': require('./momentum')
```
Comment (DeviaVir) [inline lib/_codemap.js]: indenting issue here

**Comment by DeviaVir** [comment]
I like it as a simple strategy, but averaging 1500 trades/day is not very realistic.

**Comment by Shawn8901** [comment]
The period length of the example was at 15s.

An example with Period of 5minutes and momentum_size of 4

>end balance: 1131.22547464 (13.12%)
>buy hold: 750.18837185 (-24.99%)
>vs. buy hold: 50.79%
>529 trades over 9 days (avg 58.77 trades/day)
>win/loss: 92/173
>error rate: 65.28%
>wrote simulations/sim_result_gdax.BTC-EUR_180117_210201_UTC.html

**Review by DeviaVir — APPROVED** [review]

**Comment by hig-big** [comment]
There is a problem. while other strategies are working fine, momentum is selected error; 

"zenbot/node_modules/timebucket/lib/bucket-size.js:15
  if (!match) throw new Error('invalid bucket size spec: ' + spec);
              ^

Error: invalid bucket size spec: undefined
    at BucketSize.parse (/home/zenbot/node_modules/timebucket/lib/bucket-size.js:15:21)
    at new BucketSize (/home/zenbot/node_modules/timebucket/lib/bucket-size.js:5:21)
    at Bucket.resize (/home/zenbot/node_modules/timebucket/lib/bucket.js:30:14)
    at Command.<anonymous> (/home/zenbot/commands/trade.js:345:32)
    at Command.listener (/home/zenbot/node_modules/commander/index.js:315:8)
    at emitTwo (events.js:126:13)
    at Command.emit (events.js:214:7)
    at Command.parseArgs (/home/zenbot/node_modules/commander/index.js:653:12)
    at Command.parse (/home/zenbot/node_modules/commander/index.js:475:21)
    at /home/zenbot/zenbot.js:39:11
    at /home/zenbot/boot.js:24:7
    at f (/home/zenbot/node_modules/once/once.js:25:25)
    at Glob.<anonymous> (/home/zenbot/node_modules/glob/glob.js:151:7)
    at emitOne (events.js:116:13)
    at Glob.emit (events.js:211:7)
    at Glob._finish (/home/zenbot/node_modules/glob/glob.js:197:8)
    at done (/home/zenbot/node_modules/glob/glob.js:182:14)
    at Glob._processGlobStar2 (/home/zenbot/node_modules/glob/glob.js:637:12)
    at /home/zenbot/node_modules/glob/glob.js:626:10
    at RES (/home/zenbot/node_modules/inflight/inflight.js:31:16)
    at f (/home/zenbot/node_modules/once/once.js:25:25)
    at lstatcb_ (/home/zenbot/node_modules/glob/glob.js:519:7)

**Comment by them4a1** [comment]
this is not a development proposal but a usage value proposition: it is difficult to catch down-up trends in the short-term (daily-hourly trade). more exchange fees also appears when you trade high in the momentum short-term. so cci_srsi seems to be more efficient.

**Pull request merged**